src/popover/types.js

```diff
@@ -47,14 +47,7 @@ export type OverridesT = {
 };
 
 // Basically React.Node minus React.Portal and Iterable
-export type ChildT =
-  | void
-  | null
-  | boolean
-  | number
-  | string
-  // eslint-disable-next-line flowtype/no-weak-types
-  | React.Element<any>;
+export type ChildT = void | null | boolean | number | string | React.Element<*>;
```

Comment:
wonder if this can be replaced with `React.Node` type

Reply:
check out this comment above: https://github.com/uber-web/baseui/pull/548/files/1b63648e8bb4a89627b37fbc5fc6b71d86828eaf#diff-44ceff034e5c3079fbb15abc1ced9f5eR49 looks like dev explicitly wanted to leave some options out